x-pack/plugins/observability_solution/inventory/common/utils/get_identity_fields_values.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENTITY_IDENTITY_FIELDS } from '@kbn/observability-shared-plugin/common';
+import { Entity } from '../entities';
+
+export function getIdentityFieldValues({ entity }: { entity: Entity }) {
+  const mapping: string[] = [];
+
+  const identityFields = entity[ENTITY_IDENTITY_FIELDS];
+
+  if (identityFields) {
+    const fields = Array.isArray(identityFields) ? identityFields : [identityFields];
```

Comment:
you could just:
```suggestion
    const fields = [identityFields].flat();
```

Reply:
nice thanks for the suggestion 🔥

Reply:
https://github.com/elastic/kibana/blob/b594c5a304e6b14098ecc4c9a2335895e8b86c89/x-pack/plugins/observability_solution/inventory/common/utils/parse_identity_field_values_to_kql.ts#L17